### PR TITLE
Peer Review and Fixes (ejaronne)

### PR DESCRIPTION
**Pull request by aaronlippold**
Added <% elsif ENV['BASELINE'].nil? %>
as needed to default of Moderate is the BASELINE environment variable is not supplied.

### CODE REVIEW

File: controls/overlay.rb [hunk 1]
```diff
@@ -7,25 +7,14 @@
     tag nist: ["SC-45 (1)"]
   end
```
Comment (ejaronne) [inline controls/overlay.rb:8]: We didn't replace NIST tags in 3.1 for this overlay, and shouldn't now?

File: inspec.yml [hunk 2]
```diff
@@ -365,6 +369,4 @@ inputs:
       - "V-93567" # SI-02(02)
       - "V-93563" # SI-16
       - "V-93565" # SI-16
```
Comment (ejaronne) [inline inspec.yml:371]: Add to Low baseline:
      - "V-92975" # AC-02(02)
      - "V-92977" # AC-02(02)

Comment (karikarshivani) [reply inspec.yml:373]: Done! Good catch - thanks!
[Commit reference](https://github.com/CMSgov/cms-ars-5.0-microsoft-windows-server-2019-stig-overlay/pull/2/commits/d784d78edf55bd449b351557aee0acd333f8fe98)

**Comment by karikarshivani** [comment]
Fixes #3

**Comment by ejaronne** [comment]
Looking closely at ARS 5, it appears that PasswordHistorySize *is* in play for all baselines.

This affects V-93479

According to ARS 5, the input password_history_size should be:
6 for Low, Moderate
12 for High
and "resuse is disallowed" for HVAs: but the maximum history capable in Windows is 24, so 24 for HVAs.

https://learn.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/enforce-password-history

**Comment by ejaronne** [comment]
inspec.yml logic looks good. 
N/A controls look good also.

**Comment by karikarshivani** [comment]
> Looking closely at ARS 5, it appears that PasswordHistorySize _is_ in play for all baselines. (IA-05(01))
> 
> This affects V-93479
> 
> According to ARS 5, the input password_history_size should be: 6 for Low, Moderate 12 for High and "re-use is disallowed" for HVAs: but the maximum history capable in Windows is 24, so 24 for HVAs.
> 
> https://learn.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/enforce-password-history

Thank you for pointing this out! This was a tricky control so good catch!
I pushed the pending change of adding the input override with relevant values: [commit reference](https://github.com/CMSgov/cms-ars-5.0-microsoft-windows-server-2019-stig-overlay/pull/2/commits/5396f53f2294bdeb3113038f6e17a58bd6a97f14)

**Pull request merged**